lms/djangoapps/commerce/api.py

```diff
@@ -60,26 +60,52 @@ def get():
             }
             url = '{base_url}/orders/{order_number}/'.format(base_url=self.url, order_number=order_number)
             return requests.get(url, headers=headers, timeout=self.timeout)
+
+        data = self._call_ecommerce_service(get)
+        return data['number'], data['status'], data
+
+    def get_processors(self, user):
+        """
+        Retrieve the list of available payment processors.
+
+        Returns a list of strings.
+        """
+        def get():
+            """Internal service call to retrieve the processor list. """
+            headers = {
+                'Content-Type': 'application/json',
+                'Authorization': 'JWT {}'.format(self._get_jwt(user))
+            }
+            url = '{base_url}/payment/processors/'.format(base_url=self.url)
+            return requests.get(url, headers=headers, timeout=self.timeout)
+
         return self._call_ecommerce_service(get)
 
-    def create_order(self, user, sku):
+    def create_basket(self, user, sku, payment_processor=None):
         """
-        Create a new order.
+        Create a new basket and immediately trigger checkout.
+
+        Note that while the API supports deferring checkout to a separate step,
+        this client does not currently need that capability, so that case is
+        not supported.
```

Comment:
It may also be worth noting here that in addition to deferred checkout, the API supports adding more than one product at a time to the user's basket.